src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -23,16 +23,21 @@ using RawSpeechInput = std::vector<float>;
  *
  * @param m_tokenizer tokenizer
  */
-class OPENVINO_GENAI_EXPORTS ChunkStreamerBase : public StreamerBase {
+class OPENVINO_GENAI_EXPORTS ChunkStreamerBase {
```

Comment:
why inheritance is removed? 
On current master users could create `ChunkStreamerBase` derived class and pass to `LLMPipeline::generate`, right?

I would deprecate the whole class and made it:
```c++
[[ deprecated ]]
using ChunkStreamerBase = StreamerBase;
```

Reply:
This PR adds `put(const std::vector<int64_t> tokens)` to `StreamerBase` class. With inheritance I think this is confusing to have virtuals `put(const std::vector<int64_t> tokens)` and `put_chunk` at the same time. I think of this as a breaking change.

Reply:
```
[[ deprecated ]]
using ChunkStreamerBase = StreamerBase;
```
I think this is a breaking change as user should rename `put_chunk` -> `put`.